sections/introduction.include

```diff
@@ -880,7 +880,7 @@
       bizarre and convoluted error handling rules, user agents are permitted to fail whenever
       encountering a <a for="parser">parse error</a>.
   : Errors where the error-handling behavior is not compatible with streaming user agents
-  :: Some error-handling behavior, such as the behavior for the <code>&lt;table>&lt;hr>...</code>
+  :: Some error-handling behavior, such as the behavior for the <{table}><{hr}>...
```

Comment:
This change removes the angle-brackets which are necessary as context in the text... We can still link the name of the elements, we just need to include the angle-brackets as well.